bull/email-queue/consumer.js

```diff
@@ -20,10 +22,10 @@ EmailQueue.on("active", (job, jobPromise) => {});
 EmailQueue.on("completed", (job, result) => {});
 
 EmailQueue.process((job, done) => {
-  if (job.data.email != "") {
+  if (job.data.userName != "") {
     const mailOptions = {
-      from: process.env.email, // sender address
-      to: job.data.email, // list of receivers
+      from: process.env.TOOL_FORGE_EMAIL, // sender address
+      to: `${job.data.userName}@toolforge.org`, // list of receivers
```

Comment:
@okerekechinweotito Why are we changing the "to" email address to the toolforge.org domain? Shouldn't this be the email address that is collected during the upload? I'm guessing not everyone uploading will have a Wikitech account.

Reply:
@coderwassananmol 
I have made the requested changes